Sources/RealDeviceMapLib/Misc/Localizer.swift

```diff
@@ -16,7 +16,7 @@ public class Localizer {
     public static var locale = "en" {
         didSet {
             global.load()
-            global.loadTranslationsFromRepo(language: Localizer.locale)
+            global.loadWeatherTypes()
```

Comment:
I think you mixed up those methods? `loadWeatherTypes()` is the wrong to keep?! :)